src/content/index.md

```diff
@@ -1,30 +1,35 @@
+- **Develop in Production-like Environments**: With Okteto, your development environment is a replica of your production environment. Featuring instantaneous Code Sync and Live Updates, experience the magic of seeing your code changes reflected instantly in your cloud environment. This allows for rapid testing and iteration without the need for rebuilds or redeployments, ensuring that your applications run just as smoothly in development as they do in production.
+![Developing with Okteto Example](../../static/img/dev-environment-example.jpg)
 
-## Preview Environments
-Using Okteto, you can configure [Preview Environments](preview/overview.mdx) for your code repositories. Every pull request will be deployed and will generate its own sharable URL. This helps teams browse the changes in a pull request before it gets merged without having to take care of any deployment-related setup. These URLs can also be shared with other stakeholders so the review process isn't just limited to developers.
+- **Okteto Manifest simplifies environment automation**: Utilize the [Okteto Manifest](manifest/overview-manifest.mdx) to define and configure your development environments and applications declaratively. This powerful feature ensures consistent and reproducible environments across your team, tailored to your projects' needs. Use the [Okteto Catalog](deploy/deploy-from-catalog.mdx) to create a collection of ready-to-use applications for your development team.
+![Platform team using Okteto diagram](../../static/img/manifest.jpg)
 
-You can learn how to configure Preview Environments [here](preview/overview.mdx).
+## Key Features
 
-## Why Okteto?
+### Development Environments
+Okteto's [Development Environments](deploy/development-environments.mdx) enable you to deploy and develop applications directly in the cloud with a single [CLI command](core/using-okteto-cli.mdx). Write code locally on your machine and view your changes live, deployed in a managed Kubernetes cluster **as soon as you hit save**! You don't have to spend time configuring anything to do this.
 
-Okteto was built for developers, by developers. We understand how frustrating it is to deal with the modern-day challenges of cloud-native development.
+### Automated Preview Environments
+Collaborate and share your progress with ease. Okteto's [Preview Environments](preview/overview.mdx) automatically generate a unique, shareable version of your application for each pull request, making code reviews and stakeholder feedback a breeze.
```

Comment:
I would also mention that you can also run your e2e tests against the preview environment.